README.md

```diff
@@ -1,28 +1,65 @@
 # XMTP-MEMO-JS
 
-**XMTP client SDK for Pre-Registration messaging**
+![Status](https://img.shields.io/badge/Project_Status-Developer_Preview-red)
 
-`xmtp-memo-js` provides a TypeScript implementation of an XMTP memo client.
-Build with `xmtp-memo-js` to send 'messages' to a blockchain wallet address which does not have published encryption keys on the network. `xmtp-memo-js` supports use-cases where content needs to be sent to a account which has not registered yet.
+## XMTP client SDK for pre-registration messaging
 
-**What is a memo?**
+`xmtp-memo-js` provides a TypeScript implementation of an XMTP memo client.
 
-Memo's are similar to XMTP Messages however they differ substantively.
+With XMTP, a blockchain account address must be registered on the XMTP network to send and receive end-to-end encrypted messages. Build your app with `xmtp-memo-js` to enable a registered address to start communicating with an unregistered address by sending a memo instead of a message.
 
-## Security Notice
+:warning: :warning: :warning: `XMTP:Memos` are available as an early Developer Preview. :warning: :warning: :warning: <br>
+The memo API will change without warning and may not be backward compatible.
 
-`xmtp-memo-js` uses [Lit protocol](https://developer.litprotocol.com/) to secure memos, and do not carry the same security characteristics as XMTP messages.
+### What is a memo?
 
-- Memos are **NOT** End-to-end encrypted. The encryption key is stored in the Lit Network and is only as secure as the escrow service.
+Memos are not the same as XMTP messages, though they are similar. For example, the payload structure of memos and messages is the same, but memos and messages are used in different circumstances.
```

Comment:
I adjusted the text here to try to clarify that memos and messages do not carry the same payloads - literally. Like the exact content in a memo is not also carried in a message. I tried to do this by saying "payload structure" - but wanted ensure that my change didn't change the original meaning and create an inaccurate statement!

Reply:
@jazzz - ah, I think I get it- thank you! What do you think of this edit below? And may I ask - what is the "Encoded Content" specification? Does this have to do with XMTP content types - or is this a more generic spec?

```suggestion
Memos are not the same as XMTP messages, though they are similar. For example, the payload structure of memos and messages is the same and they both use the "Encoded Content" specification. However, memos and messages are used in different circumstances.
```

Reply:
> What do you think of this edit below? 

I think I like the wording as is, actually. After the API goes through some feedback/testing we can return to make stronger claims about the interop between messages and memos. Currently in its untested state I'm hesitant to have aspirational docs.

By encoded content specification, I was referring to the way that data is encoded inside XTMP messages. It was an imprecise use of language as I think part of it is covered by the ContentTypes, but may include more.

Reply:
Ah, got it... I took it too far! LOL I'll delete it now. <3